scapy/sendrecv.py

```diff
@@ -233,7 +234,7 @@ def _process_packet(self, r):
                 if r.answers(sentpkt):
                     self.ans.append((sentpkt, r))
                     if self.verbose > 1:
-                        os.write(1, b"*")
+                        sys.stdout.write("*")
```

Comment:
Is there a reason why you are changing this? This will likely cause strange behavior as `os.write` is unbuffered unlike `sys.stdout.write`.

Reply:
The `os.write` output appears weirdly in UTscapy as it's not catched. It's just printed in the console instead of being added to the test log

Reply:
I understand the need but this should be discussed and not include in this PR, as this change as nothing to to with the loopback interface.

I am afraid that the sys.stdout buffering will cause strange display in interactive mode. The current best solution for UTScapy is to use `conf.verb=0`

Reply:
Reverted